src/proto/h1/role.rs

```diff
@@ -511,7 +511,7 @@ impl Http1Transaction for Server {
             encoder = match msg.body {
                 Some(BodyLength::Unknown) => {
                     if msg.head.version == Version::HTTP_10
-                        || !Server::can_chunked(msg.req_method, msg.head.subject)
+                        || !Server::can_have_content_length(msg.req_method, msg.head.subject)
```

Comment:
I *think* this specific one should still be `can_chunked`, since if this is false, the `else` branch will set chunked. I'm a little worried that this change didn't trigger one of the response body tests...

Reply:
Looking to this again, I agree. I have changed it now and I think the PR is complete.